### ISSUE TITLE
REQUEST: New membership for @maver1ck

### ISSUE THREAD

**Issue by maver1ck** [kubernetes/org#159]
### GitHub Username
@maver1ck

### Organization you are requesting membership in
@kubernetes

### Requirements
- [X] I have reviewed the community membership guidelines (https://git.k8s.io/community/community-membership.md)
- [X] I have enabled 2FA on my GitHub account (https://github.com/settings/security)
- [ ] I have subscribed to the kubernetes-dev e-mail list (https://groups.google.com/forum/#!forum/kubernetes-dev)
- [X] I am actively contributing to 1 or more Kubernetes subprojects
- [X] I have two sponsors that meet the sponsor requirements listed in the community membership guidelines
- [X] I have spoken to my sponsors ahead of this application, and they have agreed to sponsor my application

### Sponsors
- @davidkarlsen 
- @cpanato

### List of contributions to the Kubernetes project
- PRs reviewed / authored
https://github.com/helm/charts/pull/8325
https://github.com/helm/charts/pull/8320
https://github.com/helm/charts/pull/8318
https://github.com/helm/charts/pull/8138
https://github.com/helm/charts/pull/7961
https://github.com/helm/charts/pull/7960
https://github.com/helm/charts/pull/7150

- Issues responded to
https://github.com/helm/charts/issues/8316
https://github.com/helm/charts/issues/8289
https://github.com/helm/charts/issues/7978
https://github.com/helm/charts/issues/7959
https://github.com/helm/charts/issues/7936
https://github.com/helm/charts/issues/7708
https://github.com/helm/charts/issues/7691
https://github.com/helm/charts/issues/7148
https://github.com/helm/charts/issues/7056
https://github.com/helm/charts/issues/6727
https://github.com/helm/charts/issues/4472
https://github.com/helm/helm/issues/4697
https://github.com/helm/helm/issues/4240
https://github.com/helm/helm/issues/4239
https://github.com/kubernetes/kops/issues/5685
https://github.com/kubernetes/kops/issues/5587
https://github.com/kubernetes/kops/issues/5482

/area github-membership

**Comment by cblecker** [comment]
@maver1ck Are you able to substitute the helm/helm and helm/charts PRs for others actually in the kubernetes project? The @helm project/organization is now it's own separate CNCF project from the @kubernetes project :)

**Comment by maver1ck** [comment]
@cblecker not really.
Could you point to the information how to get helm membership ?

**Comment by cblecker** [comment]
I believe the process for helm charts trusted contributor status is here: https://github.com/helm/charts#trusted-collaborator

@mattfarina: is the above link the most accurate place for helm contributors to look at?

**Comment by cblecker** [comment]
Yes, but Kubernetes org membership (https://git.k8s.io/community/community-membership.md) requires active contribution to Kubernetes subprojects (helm and charts no longer qualifies for this).

**Comment by mattfarina** [comment]
@cblecker You have the right link for trusted collaborators, yes.
